### PR TITLE
chore(docs): rsdoctor add the note for CssExtractRspackPlugin

### PR DESCRIPTION
**Pull request by easy1090**
## Summary
 [Docs] Rsdoctor add the note for CssExtractRspackPlugin

## Checklist

- [ ] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

**Review by chenjiahan — COMMENTED** [review]
Should we move this note to the Rsdoctor documentation?

**Comment by easy1090** [comment]
> Should we move this note to the Rsdoctor documentation?

Did you mean moving "Use Rsdoctor" to Rspack's website or to Rsdoctor's website?

I'll also add it to Rsdoctor's website. If you were referring to the Rspack website, I think this note should be placed near the loader profile documentation for Rsdoctor.

**Comment by chenjiahan** [comment]
I mean move "this note" to the Rsdoctor website.

If Rsdoctor already provides this note, I don't recommend maintaining two copies of the same content. You can just link to the Rsdoctor documentation.

Considering that the Rsbuild documentation also has a section on Use Rsdoctor, it is unreasonable to maintain multiple copies of the same content.

**Comment by chenjiahan** [comment]
CI broken:

image

**Comment by easy1090** [comment]
> CI broken:
> 
> image

@chenjiahan  this is be ok~

File: website/docs/zh/guide/optimization/profile.mdx [hunk 1]
```diff
@@ -86,3 +86,11 @@ $ RSPACK_PROFILE=ALL rspack build
 - 使用文档：
   - Loader Analysis 使用文档可查看 [Loader Timeline](https://rsdoctor.dev/zh/guide/usage/loaders-timeline) 和 [Loader Details](https://rsdoctor.dev/zh/guide/usage/loaders-analysis)。
   - Plugins Analysis 使用文档可查看 [Plugin Analysis](https://rsdoctor.dev/zh/guide/usage/plugins-analysis)。
+
+#### 📢 注意
```
Comment (chenjiahan) [inline website/docs/zh/guide/optimization/profile.mdx:90]:
```suggestion
#### 📢 FAQ
```

File: website/docs/en/guide/optimization/profile.mdx [hunk 2]
```diff
@@ -84,3 +84,11 @@ To enable `RsdoctorRspackPlugin`, you can refer to the documentation: [Use Rsdoc
 - Usage documentation:
   - For Loader Analysis, refer to [Loader Timeline](https://rsdoctor.dev/guide/usage/loaders-timeline) and [Loader Details](https://rsdoctor.dev/guide/usage/loaders-analysis).
   - For Plugins Analysis, refer to [Plugin Analysis](https://rsdoctor.dev/guide/usage/plugins-analysis).
+
+#### 📢 Note
```
Comment (chenjiahan) [inline website/docs/en/guide/optimization/profile.mdx:88]:
```suggestion
#### FAQ
```

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**